### PR TITLE
[WIP] Add index on outing date_end attribute

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_api/issues/663

### CODE REVIEW

File: alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '8bd0478eea2a'
+down_revision = '91b1beed9a1c'
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    op.create_index(
+        'outing_date_end_idx', 'outings', ['date_end'], schema='guidebook')
```
Comment (tsauerwein) [inline alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py]: I would say I should be more something like:
```
Index(
            'ix_guidebook_òutings_date_end_and_document_id',
            date_end.desc(), document_id, postgresql_using='btree'),
```

But I don't have a database with enough data at hand to test.

Also, don't you also want to add the index to the model file?

Comment (asaunier) [inline alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py]: See also https://github.com/c2corg/v6_api/issues/663#issuecomment-299944457

> I would say it should be more something like:

Similar to the feed index on date desc + id:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/feed.py#L166-L174

Comment (tsauerwein) [inline alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py]: Yes, but maybe test if the index is used (on a database with many outings).

**Comment by cbeauchesne** [comment]
@asaunier is this PR still useful ? Do you think we should merge, or close ?

**Comment by asaunier** [comment]
I don't remember but it seems WIP and as you say in the related issue, it seems the problem (slow loading when no filter) no longer happens (?).
Perhaps we can close the PR.

**Comment by cbeauchesne** [comment]
Ok, I close it. I confirm that I did not noticed any slowness on this kind of request since a while. Thank you

**Pull request closed without merging**